### PR TITLE
Sound

### PR DESCRIPTION
**Pull request by fenichelar**

### CODE REVIEW

File: src/agents/Ship.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ namespace Game {
         this.ticksSinceLastFire = 0;
         // disabled for now because the current iteration is too obnoxious
```
Comment (BenTheElder) [inline src/agents/Ship.ts]: we should remove this comment

File: src/agents/Ship.ts [hunk 2]
```diff
@@ -156,7 +156,7 @@ namespace Game {
...
         // TODO: re-visit audio
```
Comment (BenTheElder) [inline src/agents/Ship.ts]: and this

**Comment by fenichelar** [comment]
Got them both. About to push.

**Comment by BenTheElder** [comment]
👍

**Pull request merged**